### ISSUE TITLE
Accessibility: Make Authors browse letters keyboard accessible

### ISSUE THREAD

**Issue by aparkening** [alistapart/AListApart#226]
From Andrew's accessibility audit:

On the “Authors” page (http://alistapart.com/authors), the letters to help readers jump to specific letters for authors can only be used with the mouse.  Keyboard users need to be able to access these also. 

I believe the issue is that the a elements don’t have an href attribute.

This is a WCAG 2.0 2.1.1 failure.
